### ISSUE TITLE
🚀 Feature: Explore plugin tools categories

### ISSUE THREAD

**Issue by sergeyshevch** [backstage/backstage#18636]
### 🔖 Feature description

Hello! I'm exited about the new Explore plugin and have a minor feature request. We have multiple tools internally that we want to add to Explore tools. But it will be good to divide it into various categories analytics/development/operations etc. Or event to different tabs on explore page

### 🎤 Context

For example, we want to use next categories:
- analytics
- operations
- development
- monitoring

### ✌️ Possible Implementation

category field in tool definition

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, I don't have time to work on this right now

**Comment by andrewthauer** [comment]
The tool data structure already has a `tags` attribute which could be used for a similar purpose. The thing that is missing is a way to filter, group, sort by various attributes in the UI.

**Comment by sergeyshevch** [comment]
For now, I implemented my Own Content component, which supports categories based on tags. But it looks like these fields have different purposes.

**Comment by sergeyshevch** [comment]
I guess this is still relevant. Is there anyone also need this feature?

**Comment by andrewthauer** [comment]
Yes, we currently have 2 tools tabs. One for Backstage tools and another for external tools. We also copied some of the component code as it's not exported in order to do this. I'd prefer to have a single "Tools" page that has some grouping and filtering all fed off the single tools data source. I added the `explore-backend` months ago with the intention of revisiting the UI to do this, but haven't found the time to do so. I'd still like to, but unsure when or if I'll be able to.

**Comment by andrewthauer** [comment]
Still interested

**Comment by efenner-cambia** [comment]
I'd still be interested in seeing this

**Comment by tudi2d** [comment]
Would you be interested in giving this a try & submitting a PR @efenner-cambia ?

**Comment by sergeyshevch** [comment]
Not stale

**Comment by sergeyshevch** [comment]
I'm out of capacity for this issue now. I've implemented some workaround on frontend with custom page and filtering by tags for myself but it's dirty hack and I guess that upstream implementation should be more correct

**Comment by efenner-cambia** [comment]
Not stale

**Comment by andrewthauer** [comment]
not stale

**Comment by Rugvip** [comment]
Moving this over to https://github.com/backstage/community-plugins, since that's where the explore plugin now lives
